nixos/tests/alist.nix

```diff
@@ -0,0 +1,31 @@
+import ./make-test-python.nix (
```

Comment:
I dont think you need `import ./make-test-python.nix (` when using `runTest`.

Reply:
yeah i forgot it, sorry